docs/mint.json

```diff
@@ -248,7 +248,7 @@
     "pages": [
       "api-reference/actions/list-actions-minimal",
       "api-reference/actions/get-action",
-      "api-reference/actions-controller-v-2/execute-action",
+      "api-reference/actions/execute-action",
```

Comment:
Consider adding a redirect rule in the redirects section for the old path `api-reference/actions-controller-v-2/execute-action` to the new path `api-reference/actions/execute-action` to maintain backward compatibility for existing documentation links.